lib/extensions/descendant_loader.rb

```diff
@@ -182,12 +182,14 @@ def save_cache!
       # Don't write sti_loader.yml in production, this shouldn't change from what is in the RPM
       if Rails.env.production?
         warn "\nSTI cache is out of date in production, check that source files haven't been modified"
+        warn "One file that was dirty was: #{@cache_dirty}"
       else
         cache_path.parent.mkpath
         cache_path.open('w') do |f|
           YAML.dump(cache, f)
         end
       end
+      @cache_dirty = nil
```

Comment:
Is this a bug fix, technically?

Reply:
I think technically yes, practically it probably doesn't matter.  If we ever called `save_cache!` before `at_exit` though then yes it'd be an issue